src/main/resources/application.properties

```diff
@@ -38,3 +38,7 @@ spring.datasource.writer.driver-class-name=com.mysql.cj.jdbc.Driver
 
 spring.datasource.jpa.hibernate.ddl-auto = update
 hibernate.diaplect =org.hibernate.dialect.MySQL8Dialect
+
+security.jwt.secret-key=3cfa76ef14937c1c0ea519f8fc057a80fcd04a7420f8e8bcd0a7567c272e007b
+# 1h in millisecond
+security.jwt.expiration-time=3600000
```

Comment:
Make the expiration duration the same as generated by Albatross

Reply:
done